src/components/Modals/SellCollectionModal/SellCollectionModal.tsx

```diff
@@ -9,14 +9,14 @@ import './SellCollectionModal.css'
 
 export default class SellCollectionModal extends React.PureComponent<Props> {
   handleSell = () => {
-    const { collection, metadata, onSetMinters } = this.props
-    onSetMinters(collection, setOnSale(collection, metadata.isOnSale))
+    const { collection, wallet, metadata, onSetMinters } = this.props
+    onSetMinters(collection, setOnSale(collection, wallet.chainId, metadata.isOnSale))
```

Comment:
I think that we will need to use the mapped `chainId` instead of the connected one